src/Common/Commands.ScenarioTest/ServiceManagement/ServiceManagementTests.cs

```diff
@@ -21,19 +21,16 @@
 
 namespace Microsoft.WindowsAzure.Commands.ScenarioTest
 {
-    public partial class AzureVMTests
+    public partial class ServiceManagementTests
     {
         private EnvironmentSetupHelper helper = new EnvironmentSetupHelper();
 
-        #region Get-AzureVM Scenario Tests
-
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
```

Comment:
Tests related to VM usually take some time. Can we create a separate items to track and fix it?